zhang_mcfarlane/zm_conv_momtran.F90

```diff
@@ -59,7 +59,7 @@ subroutine zm_conv_momtran_run(ncol, pver, pverp, &
    real(kind_phys), intent(in) :: ed(:,:)       ! Mass entraining from downdraft            (ncol,pver)
    real(kind_phys), intent(in) :: dp(:,:)       ! Delta pressure between interfaces         (ncol,pver)
    real(kind_phys), intent(in) :: dsubcld(:)       ! Delta pressure from cloud base to sfc  (ncol)
-   real(kind_phys), intent(in) :: dt                   !  time step in seconds : 2*delta_t
+   real(kind_phys), intent(in) :: dt
```

Comment:
can we put the `!  time step in seconds` comment back?

Reply:
That's fine by me @cacraigucar